@xen-orchestra/fs/src/abstract.js

```diff
@@ -435,6 +439,10 @@ export default class RemoteHandlerAbstract {
     return {}
   }
 
+  async _lock(path: string): Promise<Function> {
+    throw new Error('Not implemented')
```

Comment:
This default implementation will break backups for some handlers.

I think the default should simply be noop.